handler_templates.ts

```diff
@@ -13,22 +13,135 @@ import {isArray} from './schema';
 import {isObject} from './schema';
 import {withQueryParams} from './helpers/url';
 
-type ParamMapper<T> = (val: T) => T;
-
+/**
+ * Configuration for how to construct an HTTP request for a code-free formula definition
+ * created using {@link makeTranslateObjectFormula}.
+ *
+ * @example
+ * ```
+ * coda.makeTranslateObjectFormula({
+ *   name: "FetchWidget",
+ *   description: "Fetches a widget.",
+ *   parameters: [
+ *     coda.makeParameter({type: coda.ParameterType.String, name: "id"}),
+ *     coda.makeParameter({type: coda.ParameterType.String, name: "outputFormat"}),
+ *   ],
+ *   request: {
+ *     method: "GET",
+ *     url: "https://example.com/api/widgets/{id}",
+ *     nameMapping: {outputFormat: "format"},
+ *     transforms: {
+ *       format: function(value) {
+ *         return value.toLowerCase();
+ *       },
+ *     },
+ *     queryParams: ["id", "format"],
```

Comment:
Should "id" be in this list, given that it's used in the URL and not passed as a query param?

Reply:
Ah yep good catch.